src/prototree/models.py

```diff
@@ -338,12 +344,12 @@ def explain(
         rationalizations.
 
         :param x: tensor of shape (batch_size, n_channels, w, h)
-        :param sampling_strategy:
+        :param strategy:
 
         :return: predicted logits of shape (bs, k), node_probabilities, predicting_leaves, leaf_explanations
```

Comment:
predicting_leaves is optional but leaf_explanations is not. Is that correct?

Reply:
Sorry, I realize my type annotations and documentation weren't clear here, I've now updated them. Basically, for now the explanations can only be done for single leaf strategies, so both `predicting_leaves` and `leaf_explanations` are always present.